src/Forms/Shared/Samples/Map/ChangeBasemap/ChangeBasemap.xaml

```diff
@@ -11,10 +11,13 @@
             <RowDefinition Height="auto" />
         </Grid.RowDefinitions>
         <esriUI:MapView x:Name="MyMapView" />
-        <toolkit:BasemapGallery x:Name="MyBasemapGallery" BasemapSelected="BasemapSelected" />
-        <Button x:Name="changeBasemapButton"
+        <toolkit:BasemapGallery x:Name="MyBasemapGallery"
+                                BackgroundColor="White"
+                                GeoModel="{Binding Source={x:Reference MyMapView}, Path=Map}"
+                                Opacity="0.85" />
```

Comment:
Opacity shouldn't be set this way. It ends up being inherited by subcomponents of the view, which leads to unexpected behavior with the templates. I think the UI is clear enough with the toggle button.